process/transaction/interceptedTransaction_test.go

```diff
@@ -503,3 +503,105 @@ func TestNewInterceptedTransaction_ScTxDeployRecvShardIdShouldBeSendersShardId(t
 	assert.Equal(t, uint32(1), txIntercepted.RcvShard())
 	assert.Equal(t, uint32(1), txIntercepted.SndShard())
 }
+
+func TestNewInterceptedTransaction_GetNonce(t *testing.T) {
+	t.Parallel()
+
+	nonce := uint64(1)
+
+	tx := &dataTransaction.Transaction{
+		Nonce:     nonce,
+		Value:     big.NewInt(2),
+		Data:      "data",
+		GasLimit:  3,
+		GasPrice:  4,
+		RcvAddr:   recvAddress,
+		SndAddr:   senderAddress,
+		Signature: sigOk,
+	}
+
+	txi, err := createInterceptedTxFromPlainTx(tx)
```

Comment:
Should not retest here that err is nil and txi is not nil and so on. You should have another tests just for that.

Reply:
Removed that checks because already exists tests for that.